config/routes.rb

```diff
@@ -5,7 +5,14 @@
   get :sha, controller: :heartbeat
   get :reporting, controller: :reporting
 
+  # NOTE: this has questionable existance.
+  # Why do we need ui?
+  # When we have that https://github.com/alphagov/tech-docs-gem
   mount OpenApi::Rswag::Ui::Engine => "/api-docs"
+  # NOTE: this has questionable existance.
+  # Why do we need api?
+  # When it seems to be a static file server...
+  # That only works after `bundle exec rake rswag:specs:swaggerize`
```

Comment:
if we dont actual use the ui is there away to 

`bundle exec rake rswag:specs:swaggerize`
which uses really just generates `swagger/public_v1/api_spec.json`
and then server is via `/api-docs/public_v1/api_spec.json`

Reply:
the command needs to be fired at some point to generate the new open api spec if there have been any changes. this could be made as part of the build process